internals/cli/cli.go

```diff
@@ -338,7 +344,7 @@ func errorToMessage(e error) (normalMessage string, err error) {
 		}
 	case client.ErrorKindSystemRestart:
 		isError = false
-		msg = "pebble is about to reboot the system"
+		msg = fmt.Sprintf("%s is about to reboot the system", cmd.ProgramName)
```

Comment:
I would probably use `DisplayName` here, as it's really talking about the product, not the CLI command.